Dockerfile

```diff
@@ -1,31 +1,28 @@
-ARG base_image=ghcr.io/alphagov/govuk-ruby-base:3.1.2
-ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:3.1.2
+ARG ruby_version=3.1.2
+ARG base_image=ghcr.io/alphagov/govuk-ruby-base:$ruby_version
+ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:$ruby_version
 
-FROM $builder_image AS builder
 
-WORKDIR /app
+FROM $builder_image AS builder
 
-COPY Gemfile* .ruby-version /app/
+ENV GOVUK_NOTIFY_API_KEY=unused
 
+WORKDIR $APP_HOME
+COPY Gemfile* .ruby-version ./
 RUN bundle install
-
-COPY . /app
-
-# This is not a valid key but it needs to be in the correct format for the ruby client
-ENV GOVUK_NOTIFY_API_KEY=test-b56ea330-006a-459b-8af3-8015bb51a0e7-77b95465-3c02-4b09-9371-44e33cbdf9c6
```

Comment:
is this comment still valid? Does the dummy value need be a specific format still?

Reply:
Oh good question - forgot to mention that I'd tested this. Works just fine with the shorter (and less weirdly-specific-looking) dummy value.